contracts/interfaces/ILightClient.sol

```diff
@@ -21,18 +21,14 @@ import {Ics23Proof} from "./IProofVerifier.sol";
 enum LightClientType {
     SimTestLightClient, // Note: not deployed on any mainnets
     OptimisticLightClient, // Our native opstack light client
-    SequencerLightClient // Our native sequencer light client
+    SequencerLightClient, // Our native sequencer light client, which does not check l1 origin check to cut down on
```

Comment:
Can we/should we combine these and use a bool at deploy time to configure L1 checks?

Reply:
SequencerLightClient and ReOrgResistantSequencerLightClient